components/function-controller/internal/controllers/serverless/suite_test.go

```diff
@@ -22,6 +27,10 @@ var config FunctionConfig
 var resourceClient resource.Client
 var testEnv *envtest.Environment
 
+const (
+	testNamespace = "tutaj"
```

Comment:
not a fan of using non-english words in code ;) just a loose suggestion to change it to metasyntactic variable